style/layers.js

```diff
@@ -29,5 +29,7 @@ var americanaLayers = [
 
   layerMotorwayLabel,
 
-  layerHighwayShieldInterstate,
+  layerHighwayShieldInterstate12,
+  layerHighwayShieldInterstate3,
+  layerHighwayShieldInterstate4,
```

Comment:
Do these numbers refer to the number of characters in the highway shield?

Reply:
Okay yeah I see it in the other file, its based on `["==", "ref_length", X]`

Reply:
Correct!  You can do things like resize the shields or change the fonts when there's more characters on the shield.  It's also a great candidate for looking to see if we can do a better job at avoiding copy/paste :-/